### PR TITLE
Devtunnel no proxy

### PR DESCRIPTION
**Pull request by joostd**
This PR builds on https://github.com/YubicoLabs/passkey-workshop/pull/44 and replaces cloudflare with ms devtunnel.
It simplifies deployments because:
- tunnel endpoints expire after 30 days instead of after every restart. This means we no longer need separate profiles for docker compose to keep the tunnel running
- devtunnel can tunnel multiple ports. This means we no longer need to deploy a proxy container.

Some downsides:
- users need to authenticate first, using a MS or github account
- devtunnel does not run in a container like cloudflare did, so this seems less secure. However, because one needs to sign in first there is less risk of exposing services to unwanted visitors
- devtunnel is still new, so breaking changes are more likely.
-

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -5,37 +5,57 @@ This directory is an alternative to the ../scripts directory and can be used to
...
+
+4. Point your browser to
+
+	http://localhost:3000/
+
+# Deploy using a tunnel
 
 For mobile we need to expose the docker containers to the Internet in order to use them from a mobile phone.
-This deployment is similar the the web-deployment, except that services are exposed on an HTTPS URL instead of localhost.
+This deployment is similar the the localhost deployment, except that services are exposed on an HTTPS URL instead of localhost.
```
Comment (dmennis) [inline deploy/README.md:42]: Remove extra 'the'

File: deploy/README.md [hunk 2]
```diff
@@ -5,37 +5,57 @@ This directory is an alternative to the ../scripts directory and can be used to
...
 
-This document describes how to manually deploy the web and mobile environments, but this can be done automatically with the deploy script:
+This document describes how to manually deploy the complete workshop environment, but this can also be done automatically with the deploy scripts:
 
-- cloudflared.sh - for a proxied web and mobile deployment running over a cloudflare tunnel
+- localhost.sh - for a deployment running everything on localhost
+- devtunnel.sh - for a deployment running over a devtunnel (for access over the Internet using for instance a mobile phone)
 
-# Deploy for mobile
+# Deploy on localhost
```
Comment (dmennis) [inline deploy/README.md:15]: // Proposed deployment instructions for WEB only deployment
Deploy Web ONLY (running localhost)
1. Install Docker
2. Install devtunnel
3. logon to devtunnel first ('devtunnel user login')
    log into MS or Github via browser and return back to console
4. Clone repo
5. cd to /deploy
6. > ./localhost.sh
7. Point your browser to
	http://localhost:3000/

File: deploy/README.md [hunk 3]
```diff
@@ -5,37 +5,57 @@ This directory is an alternative to the ../scripts directory and can be used to
...
 
 This is what the tunneled workshop network looks like:
 
 ![Workshop network](passkey-workshop.png)
 
-Note that this also deploys the web front-end in order to demonstrate copyable passkeys.
-
-Also note that this deployment currently does not use keycloak, so only the TestPanel can be used in the web application.
-
 To deploy the mobile client:
```
Comment (dmennis) [inline deploy/README.md:48]: // Proposed deployment changes for Mobile (which also includes the web deployment)
 + Mobile deploy (using devtunnel)
1. Install Docker
2. Install devtunnel
3. logon to devtunnel first ('devtunnel user login')
    log into MS or Github via browser and return back to console
4. Clone repo
5. cd to /deploy
6. > ./devtunnel.sh
    Option: Apple Developer Team ID
7. > ./devtunnel.sh (if team ID needed to be manually added to .env)

**Review by dmennis — COMMENTED** [review]
Approved with comments

**Review by csalas-yubico — APPROVED** [review]

**Pull request merged**